app/config.go

```diff
@@ -360,7 +382,8 @@ func InitPocketCoreConfig() {
 func MustGetKeybase() kb.Keybase {
 	keys, err := GetKeybase()
 	if err != nil {
-		panic(err)
+		fmt.Println(err)
+		os.Exit(1)
```

Comment:
```suggestion
             log.Fatal(err)
```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)

the `log` package needs to also be added on this file for this suggestion to work